desktop_app/src/ui/components/Chat/ChatInput/index.tsx

```diff
@@ -524,15 +606,41 @@ export default function ChatInput({
           </div>
         )}
         <div className="relative">
+          <SlashCommandDropdown
+            suggestions={slashCommandSuggestions}
+            selectedIndex={selectedSlashCommandIndex}
+            onSelect={onSlashCommandSelect || (() => {})}
+            visible={showSlashCommandSuggestions}
+            inputRect={textareaRef.current?.getBoundingClientRect()}
+          />
+          {isCompleteSlashCommand(input) && (
+            <div 
+              className="absolute inset-0 pointer-events-none z-0 overflow-hidden"
+              style={{
+                fontSize: 'inherit',
+                fontFamily: 'inherit',
+                lineHeight: 'inherit',
+                padding: '0.75rem 1rem',
+                whiteSpace: 'pre-wrap',
+                wordWrap: 'break-word',
+              }}
+            >
+              <span className="bg-orange-400/40 dark:bg-orange-800/40 -mx-2 px-1 py-0.5 rounded text-transparent">
+                {input.trim()}
+              </span>
+            </div>
+          )}
```

Comment:
sorry to go back on this 😄 (was looking at how other apps do this, namely Slack).. in fact, let's revert this portion and just keep the styling in the chat input box untouched